doc/src/modules/vector/api/classes.rst

```diff
@@ -2,13 +2,13 @@
 Essential Classes in sympy.vector (doctrings)
 =============================================
 
-CoordSysCartesian
-=================
+CoordSystem3D
```

Comment:
If we really need to change the name, this would be better in a separate PR, because it makes it difficult to find what your PR title suggests you are changing.